userguide/storage.rst

```diff
@@ -1026,7 +1026,8 @@ any objects stored within the dataset. To remove the dataset, set
 :guilabel:`Confirm`, click :guilabel:`DELETE DATASET`, verify
 that the correct dataset to be deleted has been chosen by entering the
 dataset name, and click :guilabel:`DELETE`. When the dataset is busy, a
-force delete dialog option appears, showing what has the dataset in use.
```

Comment:
I think the complaint here was that it didn't tell what a dataset being "busy" meant.  I suggest (verify/adjust the wording):
`When the dataset has active shares or is still being used by other parts of the system, the dialog shows  what is still using it and allows forcing the deletion anyway. Caution: forcing the deletion of an in-use dataset can cause data loss or other problems.`